PillarChap/Pillar.pillar

```diff
@@ -567,26 +724,67 @@ On Pillar you can use some annotation in order to do some things.
 An annotation have this syntax:
 
 [[[
-${name:parameter=value|parameter2=value2}$
+${tag:parameter=value|parameter2=value2}$
 ]]]
 
-For now Pillar only have the inputFile annotation.
-
-!!!!InputFile annotation
+!!!! InputFile annotation
 
 This annotation allow you to include a file into your pillar file. We recommend you to add files at the root of your document, we still may need some 
 improvement in order to include file into a paragraph of in other structure.
 
-This annotation takes only 1 parameter, the path of your file from the baseDirectory (if you don't change the base directory, it's your working directory).
+This annotation takes only 1 parameter, the path of your file from the ==baseDirectory== (if you don't change the base directory, it's your working directory).
 
-Here some example:
+Here some examples:
 
 [[[
-	${inputFile:test.pillar}$
+  ${inputFile:test.pillar}$
 	
-	${inputFile:value=test.pillar}$
+  ${inputFile:value=test.pillar}$
 	
-	${inputFile:chapter2/test.pillar}$
+  ${inputFile:chapter2/test.pillar}$
+]]]
+
+!!!! Slide annotation
+@sec:slide
+
+This annotation is used to create a slide structure for a ==beamer== or a ==deck.js== export.
+The first parameter, ""title"", is a required parameter to give a title at your slide. If you want a slide without title just let a space. 
+In a future version of Pillar this  parameter should be optional if you do not want to have a title.
+You can also add a ""label"" to be able to create an internal link to your slide.
+
+Here some examples:
+
+[[[
+  ${slide: My slide}$
+	
+  ${slide:title= My slide}$
+	
+  ${slide: My slide |label=sld:mySlide}$
+]]]
+
+!!!! Columns
+
+/!\\ The column annotations works only for the beamer export on the version 0.60 of Pillar.
```

Comment:
- Use @@note instead of !.
- The column annotation*s* *currently* only work*s* with Beamer export.